src/iarc7_motion/task_command_handler.py

```diff
@@ -11,6 +11,9 @@
 from geometry_msgs.msg import Twist
 from geometry_msgs.msg import TwistStamped
 
+from iarc7_safety.iarc_safety_exception import (IARCSafetyException,
```

Comment:
No need to import `IARCSafetyException` here

Reply:
fixed